src/modules/env_var.rs

```diff
@@ -0,0 +1,49 @@
+use ansi_term::Color;
+use std::env;
+
+use super::{Context, Module};
+
+/// Creates a module with the value of the chosen environment variable
+///
+/// Will display the environment variable's value if all of the following criteria are met:
+///     - env_var.disabled is absent or false
+///     - env_var.variable is defined
+///     - a variable named as the value of env_var.variable is defined
+pub fn module<'a>(context: &'a Context) -> Option<Module<'a>> {
+    let mut module = context.new_module("env_var");
+    let module_style = module
+        .config_value_style("style")
+        .unwrap_or_else(|| Color::Black.bold().dimmed());
+
+    let env_name = module.config_value_str("variable");
+
+    // name is required
+    env_name?;
```

Comment:
Is it better to have this at the end of the `let env_name =` line?

i.e.,
`let env_name = module.config_value_str("variable")?;`

That way, `env_name` will be an `&str` rather than `Option<&str>`, and you can drop the `unwrap` from line 25.

Reply:
Another rookie mistake, sorry.